docs/data-collection.md

```diff
@@ -81,6 +81,11 @@ For the OpenShift 4 Developer Preview we will be sending back these exact attrib
   // cluster:node_instance_type_count:sum is the number of nodes
   // of each instance type and role.
   '{__name__="cluster:node_instance_type_count:sum"}',
+  //
+  // kubevirt metrics to be collected
+  //
+  // cluster:cnv_vm_status_running:sum is the total number of VM running in the cluster
+  '{__name__="cluster:cnv_vm_status_running:sum"}',
```

Comment:
This is probably ok (I assume this is a single series?), are there any error related metrics that track whether there are errors / failures with VMs?

Do you have alerts defined for VM failures on cluster today?

Reply:
It is meant to be a single series, yes. About the other questions let me forward the question to @fabiand (/cc @msivak)

Reply:
We need to review naming before I approve this (i.e. make sure we're consistent across new metrics types).

Reply:
errors/failures will definitely come in the future. Might make sense to have `running` as a label?

Reply:
@fromanirh is this now reflecting the VMIs or the VMs?

Reply:
@rmohr makes sense to use the kubevirt naming scheme (VMIs) for consistency, yes. Thanks for the warning about virt-controller, we will see the implementation later.
@smarterclayton Let me provide a bit of context. To provide this metric in cnv we are evaluating two approaches, which will influence the naming we are discussing here.
1: provide low-level metrics in kubevirt, aggregate using a record rule (https://prometheus.io/docs/practices/rules/) provided by a still-to-be-done prometheus mixin. 
Flow: kubevirt metric -> prometheus server -> prometheus record -> telemeter
We will still need to figure out how to deliver such mixin. Is this the preferred way to go? are prometheus mixins supported in openshift?
2: directly expose the final value in kubevirt. This is doable and fits our requirements about VMs count.
Flow: kubevirt metric -> prometheus server -> telemeter

This decision drives the naming scheme we are going to use (we cannot use the "cnv" prefix if we use the second approach), hence the reason I asked. Thanks for your reviews so far.